asabiris/orchestration/sendslack.py

```diff
@@ -123,3 +118,17 @@ def get_content_type(self, file_extension):
 		"""
 		content_type = mimetypes.guess_type('dummy' + file_extension)[0]
 		return content_type or 'application/octet-stream'
+
+	def _generate_error_message_slack(self, specific_error: str) -> Tuple[str, str]:
```

Comment:
this does not return tuple, am I right? You copypasted this from some copilot or chatgpt without reviewing the code